src/views/Simulation/components/Scene/Scene.js

```diff
@@ -92,7 +94,8 @@ const Scene = () => {
     ) {
       if (sceneContainerRef.current) sceneContainerRef.current.removeChildren();
       renderElements(sceneContainerRef, graphRef, setSelectedElementId, settings);
-      if (minimapContainerRef.current != null) minimapContainerRef.current?.renderElements();
+      if (layoutUpdate && sceneContainerRef.current) sceneContainerRef.current.setOrigin();
+      if (minimapContainerRef.current) minimapContainerRef.current?.renderElements();
```

Comment:
> `if (minimapContainerRef.current) minimapContainerRef.current?.renderElements();`

Since you changed this line, is the optional chaining operator `?.` necessary here? The condition already covers the check, so we can change this part too

Reply:
Yes, it is a remain after several changes, adds and re-changes...